lib/l10n/app_pt.arb

```diff
@@ -0,0 +1,112 @@
+{
+  "crypto": "Cripto",
+  "@crypto": {
+    "description": "titulo carteira de crytomoedas"
+  },
```

Comment:
Sugestão: Não é necessário colocar o "@crypto":{ .... } nos demais idiomas, apenas é necessário no teu arquivo principal que é o app_en.arb.
https://docs.flutter.dev/development/accessibility-and-localization/internationalization (Parte 4 e 5)

Reply:
retirei